includes/widgets/common.php

```diff
@@ -113,7 +113,7 @@ protected function register_controls() {
 		$this->start_controls_section(
 			'_section_style',
 			[
-				'label' => esc_html__( 'Advanced', 'elementor' ),
+				'label' => esc_html__( 'Layout', 'elementor' ),
```

Comment:
@matipojo 
All of the mess here is because of moving controls around and deleting 2 (See BC below)

Reply:
Can you extract each controls section to a separate method? so the re-order will be in the methods calling level?

Reply:
Done.